astropy/coordinates/builtin_frames/teme.py

```diff
@@ -0,0 +1,51 @@
+# -*- coding: utf-8 -*-
+# Licensed under a 3-clause BSD style license - see LICENSE.rst
+"""
+Contains the definition of the TEME frame.
```

Comment:
```suggestion
A coordinate or frame in the TEME (True Equator, Mean Equinox) system.
```
(for consistency with other frame docstrings)